cv-data-service-library/src/main/java/com/trihydro/library/helpers/CdotGisConnector.java

```diff
@@ -0,0 +1,74 @@
+package com.trihydro.library.helpers;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+
+import com.trihydro.library.service.RestTemplateProvider;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CdotGisConnector {
+  private final String baseUrl = "https://dtdapps.coloradodot.info/arcgis/rest/services/LRS/Routes/MapServer/exts/CdotLrsAccessRounded";
+
+  private RestTemplateProvider restTemplateProvider;
+
+  private final Logger logger = LoggerFactory.getLogger(CdotGisConnector.class);
+
+  @Autowired
+  public void InjectDependencies(RestTemplateProvider _restTemplateProvider) {
+    this.restTemplateProvider = _restTemplateProvider;
+  }
```

Comment:
chore: same as before. use a constructor
```suggestion
  public void CdotGisConnector(RestTemplateProvider _restTemplateProvider) {
    this.restTemplateProvider = _restTemplateProvider;
  }
```

Reply:
Addressed